redbot/cogs/warnings/warnings.py

```diff
@@ -340,33 +340,57 @@ def __init__(self, bot: Red):
             pass
         await ctx.send(_("User {user} has been warned.").format(user=user))
 
-    @commands.command()
+    @commands.command(usage="")
     @commands.guild_only()
     async def warnings(
         self, ctx: commands.Context, user: Optional[Union[discord.Member, int]] = None
     ):
-        """List the warnings for the specified user.
-
-        Omit `<user>` to see your own warnings.
-
-        Note that showing warnings for users other than yourself requires
-        appropriate permissions.
-        """
+        """Check your own warnings."""
         if user is None:
             user = ctx.author
-        else:
+        else:  # TODO: remove this else statement in a release after 3.3.0 (will allow users time start using [p]warncheck)
```

Comment:
Core cogs don’t have to respect the version guarantees and as such breaking changes can be done any time (just a FYI)

Reply:
I assume you mean the whole statement, not just the note?